README.md

```diff
@@ -9,8 +9,21 @@ Each of these Terraform projects are to be layered on top of each other, running
 
 ### [0. bootstrap](./0-bootstrap/)
 
-This stage executes the [CFT Bootstrap module](https://github.com/terraform-google-modules/terraform-google-bootstrap) which bootstraps an existing GCP organization, creating all the required GCP resources & permissions to start using the Cloud Foundation Toolkit (CFT).
-This includes; projects, service accounts and a Terraform state bucket. After executing this step, you will have the following structure:
+This stage executes the [CFT Bootstrap module](https://github.com/terraform-google-modules/terraform-google-bootstrap) which bootstraps an existing GCP organization, creating all the required GCP resources & permissions to start using the Cloud Foundation Toolkit (CFT). This includes:
+- The `cft-cloudbuild` project, which contains:
+  - Cloud Build implementation
+  - Cloud Source Repository
+  - Build pipeline
+  - Other resources
+- The `cft-seed` project, which contains:
+  - Terraform state bucket
+  - KMS configuration to encrypt the state bucket's content
+  - Custom Service Account used by Terraform to create new resources in GCP
+  - Other resources
+
+A good reason to have two separate projects here is to follow best practices. One best practice is to use custom service accounts instead of default ones. However, Cloud Build doesn't support custom service accounts yet. To avoid using the default Cloud Build service account `@cloudbuild.gserviceaccount.com` for deploying new infrastructure, this default service account is instead granted access to generate tokens over the Terraform custom service account. If using Jenkins, this limitation does not exist, because we can assign a custom service account directly to the GCE instance that runs as a Jenkins Agent.
```

Comment:
This is generally true, although I would advocate that we continue to use service account impersonation in the code as it allows for a good local development / execution experience without having to store static credentials for what is a very powerful service account. (so we should probably still support impersonation with Jenkins)